app/services/master_controller.py

```
@@ -2388,6 +2409,220 @@ async def set_trading_mode(self, mode: str) -> Dict[str, Any]:
+                "message": "✅ Operations resumed - trading active",
+                "timestamp": datetime.utcnow().isoformat()
+            }, user_id)
+            
+            return {
+                "success": True,
+                "message": "Operations resumed successfully",
+                "user_id": user_id,
+                "timestamp": datetime.utcnow().isoformat()
+            }
+            
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)